src/main/java/org/jenkinsci/plugins/authorizeproject/AuthorizeProjectProperty.java

```diff
@@ -146,43 +139,13 @@ public static void setStrategyCritical() {
     }
 
     /**
-     * If we are being deserialized outside of loading the initial jobs (or reloading) then we need to cross check
-     * the strategy permissions to defend against somebody trying to push a configuration relating to a user other
-     * than themselves.
-     *
-     * @return {@code this}
-     * @throws ObjectStreamException if the object cannot be deserialized.
-     * @see #setStrategyCritical()
-     */
-    private Object readResolve() throws ObjectStreamException {

```

Comment:
This change disturbs introducing "Don't restrict users to configuring the job" ([JENKINS-40739](https://issues.jenkins-ci.org/browse/JENKINS-40739) ). Hmm...